src/Orchard.Web/Modules/Orchard.Users/Extensions/MembershipSettingsExtensions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Orchard.Security {
+    public static class MembershipSettingsExtensions {
+        public static int GetMinimumPasswordLength(this IMembershipSettings membershipSettings) {
+            return membershipSettings.EnableCustomPasswordPolicy && membershipSettings.EnableCustomPasswordLength ? membershipSettings.MinimumPasswordLength : 1;
```

Comment:
Remove `EnableCustomPasswordLength` and use 7 as the default (because of upgrades scenario)